_docs/backenders/strategy-2021/roles/capacity.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: default
+title: Capacity
+parent: Roles
+grand_parent: Strategy 2021
+---
+
+# Capacity role definition
+
+**Current responsible**: Alicia A. Arenzana
+
+## Responsibility:
+    **Meetings and other FA:**
```

Comment:
I believe you need to remove the tab at the start of the line, otherwise this is printed like a code snippet :)